javatests/arcs/core/host/BUILD

```diff
@@ -53,23 +54,47 @@ arcs_kt_plan(
     ],
 )
 
-arcs_kt_jvm_library(
-    name = "particle",
-    testonly = 1,
-    srcs = glob(
-        ["*.kt"],
-        exclude = ["*Test.kt"],
-    ),
+arcs_kt_particles(
+    name = "pure-particles",
```

Comment:
Is this rename necessary? (i.e. I don't think it is but can't tell)

Reply:
Yes, because PurePerson depends on TestJvmProdHost, but the target that used to glob TestJvmProdHost also depends on PurePerson, so you have a cycle. I had to factor the host into its own target so PurePerson could target it. 

Alternatively, I could have made a custom annotation, but it makes sense to separate hosts and particles into different targets anyway.